docker/Dockerfile

```diff
@@ -36,29 +36,28 @@ ARG JUICEFS_MOUNT_IMAGE
 
 WORKDIR /app
 
-ENV JUICEFS_CLI=/usr/bin/juicefs
 ENV JFS_AUTO_UPGRADE=${JFS_AUTO_UPGRADE:-enabled}
 ENV JFS_MOUNT_PATH=/usr/local/juicefs/mount/jfsmount
 ENV JFSCHAN=${JFSCHAN}
 ENV JUICEFS_MOUNT_IMAGE=${JUICEFS_MOUNT_IMAGE}
 
-ADD https://github.com/krallin/tini/releases/download/v0.19.0/tini-${TARGETARCH} /tini
-RUN chmod +x /tini
-
+ADD docker/juicefs.sh /usr/local/bin/juicefs
```

Comment:
I haven't add this step to all the other dockerfiles yet, will do that after this approach has been approved